xero_files.yaml

```diff
@@ -591,9 +594,8 @@ paths:
                 example: '{
                           "19d4fc59-e799-410f-912b-03d4ab294d73": 2,
                           "82195976-5175-45d4-926e-807ff10892e7": 1,
-                          "a8547af2-2900-4879-98b8-f1a780c78feb": 0
-                }'                        
-  /Folders:
+                          "a8547af2-2900-4879-98b8-f1a780c78feb": 0}'
+/Folders:
```

Comment:
Invalid indentation. please indent the endpoints at the same level like the previous endpoints.

Reply:
This is done  at the same level like the previous endpoints.